src/components/CodeSnippet.vue

```diff
@@ -38,7 +38,6 @@
 
 <style>
 .code-snippet { @apply
-  bg-slate-700
```

Comment:
does a code snippet really not need a background?

Reply:
This is a wrapper for lang-specific code views; those have their own backgrounds already for syntax highlight contrast so this was a duplicate (and subtly incorrect) bg color.